metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -130,6 +131,14 @@ def extract_dbt_entities(
         if "identifier" in node and not load_catalog:
             name = node["identifier"]
 
+        comment = key
+
+        if (
+            key in all_catalog_entities
+            and all_catalog_entities[key]["metadata"]["comment"]
```

Comment:
I think the 'comment' key is not a required field (per https://schemas.getdbt.com/dbt/catalog/v1.json), so it may not be missing even if it's nullable. If so, should we use `.get()` just to be safe here?

Reply:
Good catch, I updated the PR to use `.get()`.

Reply:
Thanks!